build.gradle

```diff
@@ -19,7 +19,7 @@ task wrapper(type: Wrapper) {
 }
 
 buildscript {
-  apply from: "https://github.com/rosjava/rosjava_bootstrap/raw/indigo/buildscript.gradle"
+  apply from: "https://github.com/stratomda/rosjava_bootstrap/raw/indigo/buildscript.gradle"
```

Comment:
I didn't commit the package bump in the package.xml file because I don't know how the package version numbering works. Let me know when you update the package number so i can include that reference in my android_core bug (rosjava/android_core#254). Thanks. I appreciate it!